dev/_sources/auto_examples/connectivity/plot_cwt_sensor_connectivity.txt

```diff
@@ -105,294 +105,10 @@ Set parameters
     :align: center
 
 
-**Script output**:
```

Comment:
@jaeilepp it looks like you have logging at the `warning` level by default, as I do. This means the `info` logging bits won't be written out. @agramfort which way do we want it to be?